Linux Kernel files are located in /lib not /usr/lib on Alpine.

See also: https://github.com/anatol/booster/pull/132

diff -upr booster-0.7.orig/generator/config.go booster-0.7/generator/config.go
--- booster-0.7.orig/generator/config.go	2022-02-16 19:16:52.090788092 +0100
+++ booster-0.7/generator/config.go	2022-02-16 19:17:37.634293252 +0100
@@ -130,7 +130,7 @@ func readGeneratorConfig(file string) (*
 		}
 		conf.kernelVersion = ver
 	}
-	conf.modulesDir = path.Join("/usr/lib/modules", conf.kernelVersion)
+	conf.modulesDir = path.Join(imageModulesDir, conf.kernelVersion)
 	conf.debug = *debugEnabled
 	conf.readDeviceAliases = readDeviceAliases
 	conf.readHostModules = readHostModules
diff -upr booster-0.7.orig/generator/generator.go booster-0.7/generator/generator.go
--- booster-0.7.orig/generator/generator.go	2022-02-16 19:16:52.090788092 +0100
+++ booster-0.7/generator/generator.go	2022-02-16 19:17:13.324201531 +0100
@@ -58,8 +58,8 @@ const (
 )
 
 const (
-	imageModulesDir = "/usr/lib/modules/"
-	firmwareDir     = "/usr/lib/firmware/"
+	imageModulesDir = "/lib/modules/"
+	firmwareDir     = "/lib/firmware/"
 )
 
 // This is default modules list checked by booster. It either specifies a name of the module
